### ISSUE TITLE
Add icons for each tool

### ISSUE THREAD

**Issue by NathanLovato** [blackears/cyclopsLevelBuilder#28]
Icons would make the tools easier to spot. I can provide SVG files if you'd like.

**Comment by connor-lennox** [comment]
Agreed and this should be easy as it's just a matter of dropping icons on the Buttons. I'd be happy to do this - if you have those SVGs could you drop them somewhere I could access?

**Comment by blackears** [comment]
Sure. That would be helpful.  If you could attached the icons to this issue, I can get them from here.  The next release will also have tools for drawing cylinders and staircases, so I'd need icons for that too.

**Comment by NathanLovato** [comment]
Just drew tool icons. They try to follow Godot's style, just note that there are no strict coloring or shape guidelines for Godot's icons. Just the grey used below does get automatically remapped by the Godot editor when switching between light and dark themes, so it's one of the colors we want to use for this reason.

Feel free to directly make or request changes. Here's a preview:

![image](https://github.com/blackears/cyclopsLevelBuilder/assets/12694995/a91534c7-0fe6-41d0-b17f-67065e9ee109)

The source Inkscape file with all icons: ![cyclops-builder-icons](https://github.com/blackears/cyclopsLevelBuilder/assets/12694995/147c7fef-ad35-4b69-a991-0b7cf141eaf6)

And individual plain SVG files:

[cyclops-builder-icons-plain-svg.zip](https://github.com/blackears/cyclopsLevelBuilder/files/11465220/cyclops-builder-icons-plain-svg.zip)

**Comment by blackears** [comment]
Thanks.  Would you know how to set up a TextureButton so that they look like the other buttons in the editor toolbar?  Also, they might need a second image for their pressed state, unless TextureButton handles that automatically for you.

**Comment by NathanLovato** [comment]
You can use regular Button nodes, they have an `icon` property for this, and unlike TextureButton, it'll automatically handle the pressed state on the same model as other Godot buttons. Will PR this in a second

**Comment by blackears** [comment]
Thanks.  Will close as I don't think I need more icons for now.

**Comment by blackears** [comment]
Hi Nathan. I've added some new icons, but they're not displaying squarely in their buttons like your icons do.  They're also shrink down.  Do you know how I can get my new icons to display like yours?

**Comment by NathanLovato** [comment]
I cloned the latest version, however in Godot 4.2.1 I'm getting errors with the plugin, in particular it's missing/not registering the CyclopsAutoload (autoload I suppose).

So, I checked in the toolbar scene and couldn't see what might be causing stretching there. Could you please help me make the addon load without errors, so I can check what's happening in the context of the Godot editor?

**Comment by blackears** [comment]
What errors are you getting?  There is a bug in Godot where you need to open the project twice to have it compile properly.  If you close Godot and reopen it, does the error go away?

**Comment by blackears** [comment]
Thanks.  I've updated the icons.  I'll look into the autoload issue you mentioned.
